components/Map/Map.tsx

```diff
@@ -28,7 +28,8 @@ const Map: React.FC<MapProps> = ({ manifests }) => {
   const [bounds, setBounds] = useState<Leaflet.LatLngBoundsExpression>(defaultBounds);
 
   useEffect( () => {
-    setBounds(getBounds(manifests));
+    const manifestBounds = getBounds(manifests);
+    setBounds(manifestBounds.length > 0 ? manifestBounds : defaultBounds);
```

Comment:
`bounds` by default already is set to `defaultBounds`, it may be just more straightforward to reset it if necessary.

```suggestion
    const manifestBounds = getBounds(manifests);
    manifestBounds.length > 0 && setBounds(manifestBounds);
```